pkgs/servers/emby/default.nix

```diff
@@ -1,20 +1,20 @@
-{ stdenv, fetchurl, pkgs, unzip, sqlite, makeWrapper, mono46, ffmpeg, ... }:
+{ stdenv, fetchurl, pkgs, unzip, sqlite, makeWrapper, mono, ffmpeg, ... }:
 
 stdenv.mkDerivation rec {
   name = "emby-${version}";
-  version = "3.4.1.0";
+  version = "3.5.2.0";
 
   src = fetchurl {
-    url = "https://github.com/MediaBrowser/Emby/releases/download/${version}/Emby.Mono.zip";
-    sha256 = "08jr6v8xhmiwbby0lfvpjrlma280inwh5qp6v4p93lzd07fjynh5";
+    url = "https://github.com/MediaBrowser/Emby.Releases/releases/download/${version}/embyserver-mono_${version}.zip";
```

Comment:
Could you switch this to `fetchFromGitHub`?

Reply:
I think `fetchFromGitHub` only works for git repositories. This is a binary artifact created from the build.

Reply:
Then why are we fetching a binary artifact? Can we not build directly from source?

Reply:
I don't think emby is [publishing their build process](https://emby.media/community/index.php?/topic/51614-instructions-to-build-netcore-version-from-source/?p=496010).

Reply:
I think that is pretty user-hostile. But since we can't fix that and there is currently no way to mark a package as binary, please

- set `preferLocalBuild = true` to tell nix not to build this on hydra (as there is no building involved)
- add a comment explaining why we're fetching the binary release with the link you gave here

Reply:
Maybe [this](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=emby-server-git) is it though I think this will be blocked by us not having msbuild.